cmake/FindLAPACK.cmake

```diff
@@ -0,0 +1,430 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
Where did you copy this from? Why not use the one distributed with cmake? https://github.com/Kitware/CMake/blob/master/Modules/FindLAPACK.cmake

Reply:
They are copied from latest cmake repo with minor modification. And it is needed for, iirc, OpenBLAS.